yarn.lock

```diff
@@ -4219,31 +4137,31 @@ __metadata:
   linkType: hard
 
 "@metamask/eth-sig-util@npm:^6.0.0":
-  version: 6.0.2
-  resolution: "@metamask/eth-sig-util@npm:6.0.2"
+  version: 6.0.0
+  resolution: "@metamask/eth-sig-util@npm:6.0.0"
   dependencies:
-    "@ethereumjs/util": "npm:^8.1.0"
-    "@metamask/abi-utils": "npm:^1.2.0"
-    "@metamask/utils": "npm:^5.0.2"
-    ethereum-cryptography: "npm:^2.1.2"
+    "@ethereumjs/util": "npm:^8.0.6"
+    bn.js: "npm:^4.12.0"
+    ethereum-cryptography: "npm:^2.0.0"
     ethjs-util: "npm:^0.1.6"
     tweetnacl: "npm:^1.0.3"
     tweetnacl-util: "npm:^0.15.1"
-  checksum: e6072984d888389fe746834963be55c8b58c0698fcb5bb0a68d61ff885df9c2309a50b782fc0167a75438a2a3efd67321e5ce8a4a2a81b4d70ba07f40a875c09
+  checksum: ad257798831848f3fb82aba21d6ebc0cc15114a15a20bf288c785f722157646e71dc0dd6299d948b9c2c3d14fb56fa3c527d8c22fbb081640c87d967be2a0e75
   languageName: node
   linkType: hard
 
 "@metamask/eth-sig-util@npm:^7.0.0":
-  version: 7.0.1
-  resolution: "@metamask/eth-sig-util@npm:7.0.1"
+  version: 7.0.0
+  resolution: "@metamask/eth-sig-util@npm:7.0.0"
```

Comment:
Also deprecated. We need this to resolve to 7.0.1

Reply:
unclear to me why this change happened with this version bump...?

Reply:
I think we to bump to atleast @metamask/gas-fee-controller@^10.0.1 and possible 11.0.0 to get the fix

Reply:
I don't see a 11.0.0 release

Reply:
bumped to 10.0.1 here https://github.com/MetaMask/metamask-extension/pull/21832/commits/ba19b6e8a0274e124ff3ff7e5b05beaa35221bd5